package.json

```diff
@@ -175,6 +175,7 @@
     "mini-css-extract-plugin": "1.4.1",
     "mmjstool": "github:mattermost/mattermost-utilities#40d7640e85385248773504548ef962037ac842ea",
     "nock": "13.0.11",
+    "pdf-parse": "1.1.1",
```

Comment:
Could this be moved to the `e2e/package.json` since it's only used in the E2E tests?

Reply:
Removed pdf-parse from "Dependencies" section of main package.json and added it in "devDependencies" section of e2e/package.json